BINARY.md

```diff
@@ -66,21 +66,8 @@ many types are in the subsection. Each type is encoded as:
 type := params:vec(valtype) results:vec(valtype)
 ```
 
-And value types are encoded as:
-
-```
-valtype := 0x00     # string
-         | 0x01     # s8
-         | 0x02     # s16
-         | 0x03     # s32
-         | 0x04     # s64
-         | 0x05     # u8
-         | 0x06     # u16
-         | 0x07     # u32
-         | 0x08     # u64
-         | 0x09     # f32
-         | 0x0a     # f64
-```
+Value type encodings are best consulted by looking at the source, namely the
+`wit-writer` and `wit-parser` crates.
```

Comment:
:-/

Reply:
Ah I just didn't want to duplicate too much and let it get out of date because this was changing enough, if it has a period of stability seems fine to document though!